### PR TITLE
Add negative GBC and accurace color G&W palettes

### PR DESCRIPTION
**Pull request by Ninoh-FOX**

### CODE REVIEW

File: gnuboy-go/components/gnuboy/palettes.h [hunk 1]
```diff
@@ -5,14 +5,15 @@
 // Our custom colorization palettes
 static const uint16_t dmg_palettes[][4] = {
 	{ 0x6BDD, 0x3ED4, 0x1D86, 0x0860 }, // GB_DEFAULT_PALETTE
-	{ 0x67FE, 0x3F12, 0x29A0, 0x1060 }, // GB_G&W_PALETTE
+	{ 0x6BFC, 0x3B12, 0x31C8, 0x1060 }, // GB_G&W_PALETTE
```
Comment (kbeckmann) [inline gnuboy-go/components/gnuboy/palettes.h:8]: Which palette is accurate here and where do the values come from?

File: gnuboy-go/components/gnuboy/palettes.h [hunk 2]
```diff
@@ -5,14 +5,15 @@
...
 	{ 0x7FFF, 0x5AD6, 0x318C, 0x0000 }, // GB_2BGRAYS_PALETTE
 	{ 0x5BFF, 0x3F0F, 0x222D, 0x10EB }, // GB_LINKSAW_PALETTE
 	{ 0x639E, 0x263A, 0x10D4, 0x2866 }, // GB_NSUPRGB_PALETTE
 	{ 0x36D5, 0x260E, 0x1D47, 0x18C4 }, // GB_NGBARNE_PALETTE
 	{ 0x6FDF, 0x36DE, 0x4996, 0x34AC }, // GB_GRAPEFR_PALETTE
 	{ 0x6739, 0x6E6D, 0x4588, 0x1882 }, // GB_MEGAMAN_PALETTE
 	{ 0x7FBF, 0x46DE, 0x4DD0, 0x0843 }, // GB_POKEMON_PALETTE
+	{ 0x0000, 0x3220, 0x03FF, 0x7FFF }, // GB_NEGATIVE_PALETTE
```
Comment (kbeckmann) [inline gnuboy-go/components/gnuboy/palettes.h:16]: Is this something that users would want? Does it help with accessibility or is it just for fun?

File: gnuboy-go/components/gnuboy/palettes.h (hunk 1 above)
Comment (Ninoh-FOX) [reply gnuboy-go/components/gnuboy/palettes.h:8]: The colors were too warm, I realized while playing that they weren't exactly the same.  so I readjusted them, they are more "greenish"

File: gnuboy-go/components/gnuboy/palettes.h (hunk 2 above)
Comment (Ninoh-FOX) [reply gnuboy-go/components/gnuboy/palettes.h:16]: well, the gbc brought it natively, it's not that it's used much (maybe in spaceship games?) but it's for nostalgia the reason to include it

**Pull request merged**